applicationset/generators/git.go

```diff
@@ -180,9 +195,12 @@ func (g *GitGenerator) generateParamsFromGitFile(filePath string, fileContent []
 				params[k] = fmt.Sprintf("%v", v)
 			}
 			pathParamName := "path"
+			commitSHAParamName := "commitSHA"
```

Comment:
Can we move this constants out of the for loop or to the top of the function and use them instead of strings `"commitSHA"` and `"path"`? For example at line 186 and 187?